### PR TITLE
feat: added decrypt cli

### LINKED ISSUE #514: Add CLI: decrypt
- To decrypt something with a combined key (provided by oracles)

### PR DESCRIPTION
**Pull request by gyuguen**
close #514

### CODE REVIEW

**Review by H4NLee — APPROVED** [review]
LGTM
I saw the TODO, but I think it's good to keep this way. Could you explain why do you think that?

**Comment by gyuguen** [comment]
> LGTM I saw the TODO, but I think it's good to keep this way. Could you explain why do you think that?

We know the oracle endpoint from the oracle store, we can provide enough encryptedCombinedKey to fetch on our own.
I think we can get oracleAddress and get encryptionCombinedKey from it.

But it just only comfort.

File: cmd/panacead/cmd/root.go [hunk 1]
```diff
@@ -115,6 +115,7 @@ func initRootCmd(rootCmd *cobra.Command, encodingConfig params.EncodingConfig) {
 		AddGenesisWasmMsgCmd(app.DefaultNodeHome),
 		AddGenesisOracleCmd(app.DefaultNodeHome),
 		EncryptDataCmd(app.DefaultNodeHome),
+		DecryptData(app.DefaultNodeHome),
```
Comment (inchori) [inline cmd/panacead/cmd/root.go:118]: How about we change to `DecryptData` as other cmd function name?

**Review by inchori — APPROVED** [review]
LGTM!
I left a not important comment.

**Review by audtlr24 — APPROVED** [review]
lgtm

File: cmd/panacead/cmd/root.go (hunk 1 above)
Comment (gyuguen) [reply cmd/panacead/cmd/root.go]: fixed: https://github.com/medibloc/panacea-core/pull/545/commits/b432c3840b8748e98751b9b541387e82d7aaf4c4

**Pull request merged**